<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Counter v1.8</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/icons/icon24.ico</normaloff>:/img/icons/icon24.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2" colspan="2">
     <widget class="QCheckBox" name="checkLines">
      <property name="text">
       <string>Посчитать количество строк</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">QLabel {
	font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkSpaces">
      <property name="text">
       <string>Не считать пробелы</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="checkSym">
      <property name="text">
       <string>Не считать знаки</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">QLabel {
	font-size: 15px;
}</string>
      </property>
      <property name="text">
       <string>          Счётчик:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QCheckBox" name="checkWords">
      <property name="text">
       <string>Посчитать количество слов</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
